dev/benchmarks/macrobenchmarks/lib/src/animated_complex_opacity.dart

```diff
@@ -21,7 +21,13 @@ class _AnimatedComplexOpacityState extends State<AnimatedComplexOpacity> with Si
   @override
   void initState() {
     super.initState();
-    controller.forward(from: 0.0);
+    controller.repeat();
```

Comment:
This may affect the existing opacity benchmarks, I'm not sure where we note that so it doesn't surprise the sheriff at the time, but maybe this comment will help.